app/grants/management/commands/grants_cv.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""
+
+Copyright (C) 2021 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+
+from pprint import pprint
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+import requests
+from grants.models import Grant
+
+
+class Command(BaseCommand):
+
+    help = 'pulls data from https://twitter.com/austingriffith/status/1529825114597908484 and inserts into db'
+
+    def handle(self, *args, **options):
+
+        def run_query(q):
+            url = 'https://api.thegraph.com/subgraphs/name/danielesalatti/gtc-conviction-voting-rinkeby'
+            if not settings.DEBUG:
+                url = 'https://api.studio.thegraph.com/query/20308/gtc-conviction-voting-mainnet/v0.0.1'
```

Comment:
@DanieleSalatti this should point to optimism, no?

Reply:
@owocki we are going for both Optimism and Mainnet - I can switch to Optimism only if you prefer